app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -65,20 +66,22 @@
           <TextView
             android:id="@+id/chapter_summary"
             android:layout_width="0dp"
-            android:layout_height="0dp"
+            android:layout_height="wrap_content"
             android:layout_marginStart="8dp"
             android:layout_marginTop="8dp"
             android:layout_marginEnd="8dp"
+            android:layout_marginBottom="4dp"
             android:ellipsize="end"
             android:fontFamily="sans-serif"
             android:maxLines="4"
```

Comment:
This should be 3

Reply:
it should be multiple of 4? or should i make it 3